source/contributing/guidelines.rst

```diff
@@ -29,10 +29,17 @@ General steps
 
 .. note::
     Please don't submit pull requests for small, non-functional changes such as fixing typos or renaming variable names.
-    Instead, `join #sponge on IRC (irc.esper.net) <https://webchat.esper.net/?channels=sponge>`_ or 
-    `join #spongedev on IRC (irc.esper.net) <https://webchat.esper.net/?channels=spongedev>`_ and we'll change it
-    together with the other smaller changes. You can also add it to our list of 
-    `Minor Issues on GitHub <https://github.com/search?q=org%3ASpongePowered+Ongoing+Minor+Issue+List&type=Issues>`_.
+    Instead, you can:
+    
+    * `join #sponge on IRC (irc.esper.net) <https://webchat.esper.net/?channels=sponge>`_ or
+      `join #spongedev on IRC (irc.esper.net) <https://webchat.esper.net/?channels=spongedev>`_
+      and we'll change it together with the other smaller changes.
+
+    * Join our `Discord Server <https://discord.gg/PtaGRAs>`_ and post on the ``#general`` or ``#dev`` channels.
```

Comment:
https://discord.gg/sponge (here and elsewhere, I'm just going to be lazy and put it the once)